### PR TITLE
Error response

### PR DESCRIPTION
**Pull request by hwilson2563**
This addresses issue: #15 

I was able to format the error response in the last response window as so using the logic used in writing the response in the colsole: 
image

### CODE REVIEW

**Comment by hwilson2563** [comment]
Let me know if there is anything additional I need to do to this pr :) @acrosman

**Comment by acrosman** [comment]
Sorry I have been meaning to load it up and test it out but not quite getting to it. I'll admit to being a little confused about why your changes would fix it (which doesn't mean they don't, just that it isn't obvious to me when glancing at the diff). I had kinda been thinking about putting the feedback into both the raw response _and_ the message area.  But assuming this works as planned it's a major improvement over the current state. I have a pretty messed up org I can try errors against nicely so I really will try to work on this very soon.

File: main.js [hunk 1]
```diff
@@ -243,10 +243,11 @@ ipcMain.on('sf_query', (event, args) => {
   const conn = sfConnections[args.org];
   conn.query(args.rest_api_soql_text, (err, result) => {
     if (err) {
+      debugger
```
Comment (hwilson2563) [inline main.js:246]:
```suggestion
```

**Comment by hwilson2563** [comment]
> Sorry I have been meaning to load it up and test it out but not quite getting to it. I'll admit to being a little confused about why your changes would fix it (which doesn't mean they don't, just that it isn't obvious to me when glancing at the diff). I had kinda been thinking about putting the feedback into both the raw response _and_ the message area. But assuming this works as planned it's a major improvement over the current state. I have a pretty messed up org I can try errors against nicely so I really will try to work on this very soon.

@acrosman  you are completely fine, I wasn't intending to rush you, just looking to see if it was sitting due to any issues I could correct.

 I was surprised I couldn't just do a simple dot notation to the error.message, this code actually broke the display all together when initially attempting to get the message displaying. My only conclusion on why you can't see it as an object is because it is not truly an object. I can attempt to format it so it displays as an object in the last response area if you'd like. 

But overall this does work and shows the error message associated with the call both in the console and response area.

**Comment by hwilson2563** [comment]
@acrosman Would I be able to get hacktoberfest-accepted on this pr :)

**Comment by acrosman** [comment]
@hwilson2563 sorry I totally lost track of this.  Absolutely a fair request.

**Pull request merged**